From 0a159b19581b3ee5b73151efc91bcc5ef83c1ba0 Mon Sep 17 00:00:00 2001
From: Yuantian Tang <andy.tang@nxp.com>
Date: Thu, 12 Sep 2019 17:43:48 +0800
Subject: [PATCH 2/2] environment variables settings for building gpu

---
 driver/tytenv | 17 +++++++++++++++++
 1 file changed, 17 insertions(+)
 create mode 100644 driver/tytenv

diff --git a/driver/tytenv b/driver/tytenv
new file mode 100644
index 000000000..448795bee
--- /dev/null
+++ b/driver/tytenv
@@ -0,0 +1,17 @@
+
+export TOOLCHAIN=/home/tyt/bin/gcc-linaro-6.3.1-2017.05-x86_64_aarch64-linux-gnu/bin/aarch64-linux-gnu-
+export PATH=$TOOLCHAIN/bin:$TOOLCHAIN/bin/aarch64-poky-linux:$TOOLCHAIN/usr/bin:$TOOLCHAIN/usr/include:$TOOLCHAIN/usr/lib:$PATH
+export CROSS_COMPILE=/opt/fsl-imx-fb/4.9.51-mx8-beta/sysroots/x86_64-pokysdk-linux/usr/bin/aarch64-poky-linux/aarch64-poky-linux-
+export ROOTFS=/opt/fsl-imx-fb/4.9.51-mx8-beta/sysroots/aarch64-poky-linux/
+export ROOTFS_USR=$ROOTFS/usr
+export X11_ARM_DIR=$ROOTFS/usr
+export WAYLAND_DIR=/tftpboot/tangyt/wayland
+export CFLAGS="--sysroot=$ROOTFS -D__arm64"
+export LFLAGS="--sysroot=$ROOTFS"
+export PFLAGS="--sysroot=$ROOTFS"
+export BUILD_HARD_VFP=1
+export HAVE_G2D=1
+export ARCH=arm64
+export SOC_PLATFORM=freescale-ls
+export VIVANTE_SDK_DIR=/mnt/tyt/git/gpu-viv6/driver/build/sdk
+export KERNEL_DIR=/mnt/dash-lts
-- 
2.17.1

